charts/kong/values.yaml

```diff
@@ -128,6 +128,40 @@ status:
     containerPort: 8543
 
 # Specify Kong proxy service and listener configuration
+cluster:
+  # If you want to specify annotations for the Manager service, uncomment the following
+  # line, add additional or adjust as needed, and remove the curly braces after 'annotations:'.
+  annotations: {}
+  #  service.beta.kubernetes.io/aws-load-balancer-proxy-protocol: "*"
+
+  tls:
+    enabled: false
+    servicePort: 8005
+    containerPort: 8005
+    # Set a nodePort which is available if service type is NodePort
+    # nodePort: 32443
+
+  type: NodePort
```

Comment:
Why `NodePort` and not `ClusterIP`?

Reply:
Pretty much arbitrary--we use it as the default elsewhere, though I don't think we have a solid reason to do so. That default can and probably should be re-evaluated, though it's out of scope for this.

Reply:
If the `NodePort` is not used anywhere, then I strongly suggest changing this to `ClusterIP`:
* Unnecessary `NodePort`s quickly dry out a scarce resource (TCP port space) in larger clusters
* Security: a `NodePort` is an additional attack vector comparing to `ClusterIP`